### DISCUSSION TITLE
Feature Request:  Enable Greylisting for an alias

### DISCUSSION THREAD

**Discussion by ccorneli** [anonaddy/anonaddy#368] [Feature Requests]
The service I used before AA had an interesting feature that would allow you to enable 'greylisting' for a particular alias that was starting to get spam when you didn't necessarily want to disable/delete completely.  It cut down the spam significantly.

For those not familiar, the general idea is that the mail server automatically sends a temporary error message that it should try again in about 15 minutes.  But since spam bots typically just fire-and-forget and ignore responses like bounced email... they won't bother following up.  Only legitimate mail servers will have an interest to try again.  (Hopefully the spammers haven't become more sophisticated to adjust for this.)

For reference:
https://en.wikipedia.org/wiki/Greylisting_(email)
https://www.reddit.com/r/sysadmin/comments/23g65z/might_be_common_knowledge_but_holy_crap/
https://www.email-validator.net/blog/what-is-greylisting-and-how-it-stops-spam-email/

**Comment by willbrowningme** [comment]
Thanks for the suggestion, this would likely need to be implemented using Rspamd so I would need to find out a way to access a user's custom settings for AnonAddy in Rspamd when an email is received.
